dnf/cli/format.py

```diff
@@ -15,8 +15,23 @@
 # Red Hat, Inc.
 
 from __future__ import unicode_literals
+from datetime import datetime
 from dnf.pycomp import long
 
+def format_date(timestamp):
+    """Format UTC timestamp as human readable date in local timezone"""
+    if timestamp <= 0:
+        return ''
```

Comment:
Personally, I don't like this (and the same below). It should return a value (e.g. `'invalid date'`) or raise an exception. I understand that it's probably useful in your particular use case but since the function is here, it should be more general. Other may disagree with me.